spec/Powertrain/FuelSystem.vspec

```diff
@@ -40,6 +40,12 @@ TankCapacity:
   unit: l
   description: Capacity of the fuel tank in liters.
 
+FuelStatus:
```

Comment:
What is the difference to level? I would suggest when we introduce this, we remove level. Otherwise we store the same thing twice

Reply:
In the description you can notice that main difference is in unit. Question is more how and where these values would be mapped in the actual system, you could have % on much higher level like HMI and liters could be used elsewhere. @SebastianSchildt did you use level before?

Reply:
Alternatively we could consider calling it `AbsoluteLevel` (and possibly rename the other to `RelativeLevel`)

Reply:
@erikbosch that could work. FuelSystem.AbsoluteLevel + Relative